hibernate-core/src/main/java/org/hibernate/cfg/AvailableSettings.java

```diff
@@ -2556,6 +2556,23 @@ public interface AvailableSettings {
 	@Incubating
 	String PREFERRED_INSTANT_JDBC_TYPE = "hibernate.type.preferred_instant_jdbc_type";
 
+	/**
```

Comment:
Just what I said I do not want to see ;)

Reply:
I agree.

Reply:
I thought you meant the code for the preferred enum type code, but also the one for arrays? I only introduced that to allow backwards compatibility, you know, the thing we will have to keep attention to in order for EAP to get updates 😉
But seriously, I can certainly remove this as well, though I believe this particular setting is valuable.

Reply:
No EAP version has 6.0, which also means that no EAP version supports basic arrays.  Nothing to see here, move along...

Reply:
Alright, as discussed on Zulip, we will remove the setting, as we don't think anyone is relying on this and really want that particular behavior. So if someone complains later, we might re-add it, but not now.